eth-contracts/migrations/11_eth_rewards_manager_migration.js

```diff
@@ -14,7 +14,7 @@ module.exports = (deployer, network, accounts) => {
     const proxyDeployerAddress = config.proxyDeployerAddress || accounts[11]
     const guardianAddress = config.guardianAddress || proxyDeployerAddress
     const solanaRecipientAddress = config.solanaRecipientAddress || Buffer.from('0000000000000000000000000000000000000000000000000000000000000000', 'hex')
-    const antiAbuseOracleAddress = config.antiAbuseOracleAddress || accounts[12]
+    const antiAbuseOracleAddresses = config.antiAbuseOracleAddresses || [accounts[12], accounts[13], accounts[14]]
```

Comment:
can we make the corresponding contract-config update just so we don't run into confusion later? https://github.com/AudiusProject/audius-protocol/blob/db54dd412fb3ab94a5c284e69b7a17d1f9b7ab66/eth-contracts/contract-config.js#L9